src/components/LoginPage/slideComponents.js

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+
+export const Img = styled.img`
+  width: 100%;
+  height: auto;
+  filter: ${props => props.shadow ? "drop-shadow( 0px 7px 4px rgba(0, 0, 0, .3))" : "none"} ;
+`;
+
+export const Caption = styled.svg`
+  font-weight: ${props => props.fontWeight || 300};
+  position: absolute;
+  width: ${props => props.width};
+  top: ${props => props.top};
+  left: ${props => props.left};
+  right: ${props => props.right};
+  bottom: ${props => props.bottom};
+`;
+
+//aspect ratio of the slide in the layout (375px/334px)
+export const ASPECT_RATIO = 1.1;   
```

Comment:
Why not use JS to calcualte it?
`export const ASPECT_RATIO = 375 / 334;`

Reply:
Ok

Reply:
Done in commit [5c474d6](https://github.com/DarthVanger/NeuralNotes/pull/46/commits/5c474d679beb09cdf8eb5db31e9ce6cf16a6d808)